<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContourExtractorDialog</class>
 <widget class="QDialog" name="ContourExtractorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ë“±ê³ ì„  ì¶”ì¶œ ë„êµ¬ (Contour Extractor) - ArchToolkit</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_Header">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;ë“±ê³ ì„  ë°ì´í„° ì¶”ì¶œ&lt;/span&gt;&lt;/p&gt;&lt;p&gt;ìˆ˜ì¹˜ì§€í˜•ë„(DXF)ì—ì„œ ë“±ê³ ì„ ë§Œ í•„í„°ë§í•˜ê±°ë‚˜, DEMì—ì„œ ìƒˆë¡œìš´ ë“±ê³ ì„ ì„ ìƒì„±í•©ë‹ˆë‹¤.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string>background-color: #e8f5e9; padding: 10px; border-radius: 4px;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupMode">
     <property name="title">
      <string>ì¶”ì¶œ ë°©ì‹ ì„ íƒ</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_Mode">
      <item>
       <widget class="QRadioButton" name="radioDxf">
        <property name="text">
         <string>ìˆ˜ì¹˜ì§€í˜•ë„(DXF) í•„í„°ë§</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioDem">
        <property name="text">
         <string>DEMì—ì„œ ë“±ê³ ì„  ìƒì„±</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupDxf">
     <property name="title">
      <string>1. DXF ë“±ê³ ì„  í•„í„°ë§</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_Dxf">
      <item>
       <widget class="QLabel" name="label_DxfInfo">
        <property name="text">
         <string>í•„í„°ë§í•  ë ˆì´ì–´ë¥¼ ì„ íƒí•˜ê³ , ì¶”ì¶œí•  ë“±ê³ ì„  ìœ í˜•ì„ ì²´í¬í•˜ì„¸ìš”.</string>
        </property>
        <property name="styleSheet">
         <string>color: #666; font-style: italic;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listDxfLayers">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>120</height>
         </size>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRefreshLayers">
        <property name="text">
         <string>ğŸ”„ ë ˆì´ì–´ ëª©ë¡ ìƒˆë¡œê³ ì¹¨</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_ContourType">
        <property name="text">
         <string>ì¶”ì¶œí•  ë“±ê³ ì„  ìœ í˜•:</string>
        </property>
        <property name="styleSheet">
         <string>font-weight: bold; margin-top: 5px;</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_Contours">
        <item row="0" column="0">
         <widget class="QCheckBox" name="chkContour1">
          <property name="text">
           <string>ì£¼ê³¡ì„  (F0017110) - ê¸°ë³¸ ë“±ê³ ì„ </string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="chkContour2">
          <property name="text">
           <string>ê³„ê³¡ì„  (F0017111) - êµµì€ ì„ </string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="chkContour3">
          <property name="text">
           <string>ê°„ê³¡ì„  (F0017112) - íŒŒì„ </string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="chkContour4">
          <property name="text">
           <string>ì¡°ê³¡ì„  (F0017113) - ì ì„ </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btnResetFilter">
        <property name="text">
         <string>â†©ï¸ í•„í„° ì´ˆê¸°í™” (ì›ë˜ ìƒíƒœë¡œ)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupDem">
     <property name="title">
      <string>2. DEMì—ì„œ ë“±ê³ ì„  ìƒì„±</string>
     </property>
     <property name="enabled">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_Dem">
      <item>
       <widget class="QLabel" name="label_DemInfo">
        <property name="text">
         <string>DEM ë˜ìŠ¤í„°ë¥¼ ì„ íƒí•˜ê³  ë“±ê³ ì„  ê°„ê²©ì„ ì§€ì •í•˜ì„¸ìš”.</string>
        </property>
        <property name="styleSheet">
         <string>color: #666; font-style: italic;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QgsMapLayerComboBox" name="cmbDemLayer"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_Interval">
        <item>
         <widget class="QLabel" name="label_Interval">
          <property name="text">
           <string>ë“±ê³ ì„  ê°„ê²© (m):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinInterval">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_Buttons">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnRun">
       <property name="text">
        <string>ë“±ê³ ì„  ì¶”ì¶œ ì‹¤í–‰</string>
       </property>
       <property name="styleSheet">
        <string>padding: 10px 20px; font-weight: bold; background-color: #4caf50; color: white;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>ë‹«ê¸°</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgis.gui</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
